src/framework/mpas_timekeeping.F

```diff
@@ -98,18 +92,31 @@ subroutine mpas_timekeeping_init(calendar)
       if (trim(calendar) == 'gregorian') then
          TheCalendar = MPAS_GREGORIAN
 #ifndef MPAS_NO_ESMF_INIT
+#ifndef MPAS_EXTERNAL_ESMF_LIB
          call ESMF_Initialize(defaultCalendar=ESMF_CALKIND_GREGORIAN)
+#else
+         call ESMF_Initialize(defaultCalKind=ESMF_CALKIND_GREGORIAN)
+#endif
 #endif
       else if (trim(calendar) == 'gregorian_noleap') then
          TheCalendar = MPAS_GREGORIAN_NOLEAP
 #ifndef MPAS_NO_ESMF_INIT
+#ifndef MPAS_EXTERNAL_ESMF_LIB
          call ESMF_Initialize(defaultCalendar=ESMF_CALKIND_NOLEAP)
+#else
+         call ESMF_Initialize(defaultCalKind=ESMF_CALKIND_NOLEAP)
+#endif
 #endif
-!     else if (trim(calendar) == '360day') then
-!        TheCalendar = MPAS_360DAY
-!#ifndef MPAS_NO_ESMF_INIT
+     else if (trim(calendar) == '360day') then
```

Comment:
The indentation of this else if statement does not appear to match match the indentation of the else if on line 103 and the if statement on line 92.

Reply:
I've just added a commit to this PR that corrects the indentation.